app/src/main/java/com/keylesspalace/tusky/db/NotificationEntity.kt

```diff
@@ -0,0 +1,120 @@
+/* Copyright 2021 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.db
+
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.TypeConverters
+import com.keylesspalace.tusky.entity.FilterResult
+import com.keylesspalace.tusky.entity.Notification
+import com.keylesspalace.tusky.entity.Status
+import java.util.Date
+
+data class NotificationDataEntity(
```

Comment:
in the end we should give each of them a one-sentence doc explaining what they are for

Reply:
I was confused for a while before I realized that this class is not actually a DB table